libbeat/metric/system/diskio/diskstat_linux.go

```diff
@@ -104,17 +130,17 @@ func (stat *IOStat) CalcIOStatistics(counter disk.IOCountersStat) (IOMetric, err
 	result.WriteRequestCountPerSec = perSec(wrIOs)
 	result.ReadBytesPerSec = perSec(rdBytes)
 	result.WriteBytesPerSec = perSec(wrBytes)
-	result.AvgRequestSize = size
-	result.AvgQueueSize = queue
-	result.AvgAwaitTime = wait
+	result.AvgRequestSize = common.Round(size, common.DefaultDecimalPlacesCount)
```

Comment:
Not necessarily a concern, but [this article](https://www.cockroachlabs.com/blog/rounding-implementations-in-go/) does a nice job of explaining the pitfalls of rounding floats.

Reply:
Yep. In this case, that `common.Round` idiom is everywhere in beats, and it was bugging me to not have it here.

Reply:
Not for here, but it's probably worth looking at changing that; float rounding should really not happen until render time.